specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -184,67 +213,97 @@ int main()
         }
     }
 
-    // Unregisters the inproc COM Activator before exiting
-    PushNotificationManager::UnregisterActivator(token, PushNotificationRegistrationKind::ComActivator);
+    if(isComActivationSupported)
+    {
+        // Unregister the ComActivator from inproc activation
+        PushNotificationManager::UnregisterActivator(PushNotificationRegistrationActivators::ComActivator);
+    }
 
     return 0;
 }
 ```
 
 ## In this scenario, the process that Registers the Push Trigger and the process specified as the COM server are different.
+
+This only works for packaged applications that support PushTrigger and ComActivator flags in
+RegisterActivator. Out-of-process activation is not supported for unpackaged apps.
+
 Process A (Registration of the Push Trigger only):
+
 ```cpp
 int main()
 {
-    PushNotificationActivationInfo info(
-        PushNotificationRegistrationKind::PushTrigger,
-        winrt::guid("BACCFA91-F1DE-4CA2-B80E-90BE66934EC6"));
+    PushNotificationActivationInfo info = nullptr;
+    if (PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationActivators::PushTrigger))
+    {
+        // Register the ComActivator guid
+        info = PushNotificationActivationInfo(PushNotificationRegistrationActivators::PushTrigger,
+            winrt::guid("BACCFA91-F1DE-4CA2-B80E-90BE66934EC6"));
```

Comment:
If this is a sample GUID created by the application, consider hiding it here or making it invalid so people don't cut-and-paste this into their code, like:

```c++
// Be sure to create your own activator ID for each app and registration
static const winrt::guid g_myActivatorId("BACCFA91-F1DE-4CA2-xxxx-90BE66934EC6");

...

info = PushNotificationActivationInfo(PushNotificationRegistrationActivators::PushTrigger, g_myActivatorId);
```